app/decorators/authorization_request_event_decorator.rb

```diff
@@ -0,0 +1,54 @@
+class AuthorizationRequestEventDecorator < ApplicationDecorator
+  delegate_all
+
+  def user_full_name
+    return unless user
+
+    user.full_name
+  end
```

Comment:
Par curiosité, d'où vient l'attribut `user`? 

```suggestion
  def user_full_name
    user&.full_name
  end
```